### PR TITLE
[FLINK-24221][doc]Translate "JAR Statements" page of "SQL" into Chinese

### PR DESCRIPTION
**Pull request by hapihu**
Translate "JAR Statements" page of "SQL" into Chinese

https://ci.apache.org/projects/flink/flink-docs-master/docs/dev/table/sql/jar/

docs/content.zh/docs/dev/table/sql/jar.md

### CODE REVIEW

**Comment by wuchong** [comment]
cc @RocMarshal , do you have time to help reviewing this?

**Comment by RocMarshal** [comment]
@wuchong @hapihu OK. I'll check it ASAP.

File: docs/content.zh/docs/dev/table/sql/jar.md [hunk 1]
```diff
@@ -68,22 +70,22 @@ Flink SQL> REMOVE JAR '/path/hello.jar';
...
 ## REMOVE JAR
 
 ```sql
 REMOVE JAR '<path_to_filename>.jar'
 ```
 
-Currently it only supports to remove the jar that is added by the [`ADD JAR`](#add-jar) statements.
+目前只支持删除 [`ADD JAR`](#add-jar) 语句添加的 jar。
 
 ## SHOW JARS
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/table/sql/jar.md:83]: @hapihu link tag?

File: docs/content.zh/docs/dev/table/sql/jar.md [hunk 2]
```diff
@@ -68,22 +70,22 @@ Flink SQL> REMOVE JAR '/path/hello.jar';
 ADD JAR '<path_to_filename>.jar'
 ```
 
-Currently it only supports to add the local jar into the session classloader.
+目前只支持将本地 jar 添加到会话类类加载器（session classloader）中。
 
 ## REMOVE JAR
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/table/sql/jar.md:75]: link tag?

**Review by RocMarshal — COMMENTED** [review]
Thank you for your contribution @hapihu . I left two minor comments. PTAL.

File: docs/content.zh/docs/dev/table/sql/jar.md (hunk 2 above)
Comment (hapihu) [reply docs/content.zh/docs/dev/table/sql/jar.md:79]: Sorry, I thought the title was in English, so I didn't need to write the link tag.
(The link tag is automatically generated when the title is in English.)
I've added the link tag .
Please help me review it again~Thank you ~

**Review by RocMarshal — APPROVED** [review]
@hapihu Thanks for the contribution. LGTM+1.
Now ping @wuchong .

**Pull request merged**